src/routes/ratings.ts

```diff
@@ -9,4 +9,6 @@ router.post('/', RatingController.newRating);
 
 router.get('/avg/:id([0-9]+)', RatingController.getAverageRatingByServiceId);
 
+router.get('/avg', RatingController.getTopFiveAverageRatings);
```

Comment:
```suggestion
router.get('/top/:amount([0-9]+)', RatingController.getTopRatings);
```

Hvad med at man kan hente x antal med det her?

Reply:
Tjek evt hvordan man normalvis gør med REST

Reply:
måske noget med ?limit=5?offset=1 giver nr 2 til 4